lib/cli/src/extract.js

```diff
@@ -0,0 +1,88 @@
+import puppeteer from 'puppeteer';
```

Comment:
Can we pull this into a separate PR?

Reply:
sure

Reply:
https://github.com/storybookjs/storybook/pull/10146